### DISCUSSION TITLE
Implement a "searchIgnoreWords" config setting so Craft would exclude search results that contained those words

### DISCUSSION THREAD

**Discussion by monchito** [craftcms/cms#8380] [Ideas]
It would be great to have a way to add a "stop word" list to Craft to tell Craft searches to ignore any results that included those words.

[MySQL ](https://dev.mysql.com/doc/refman/5.7/en/fulltext-stopwords.html) and [PostgreSQL ](https://www.postgresql.org/docs/9.1/static/textsearch-dictionaries.html ) both have their own implementations of this, but it's not always feasible or practical for a user to be able to modify those at the server level.

**Comment by MattWilcox** [comment]
Doesn't this already exist as searchIgnoreWords in config/general.php ? Or am I misunderstanding?

'searchIgnoreWords' => array('a', 'i', 'the', 'and', 'as', 'if')

**Comment by angrybrad** [comment]
@MattWilcox Awkwardly, the config setting existed, but nothing in the code ever used it. Checked the history and it was that way all the way back to pre 1.0. ¯\_(ツ)_/¯

**Comment by MattWilcox** [comment]
PMSL! I've had a default config with that very string in since craft 1.0 too, hah!
